web/test/test_profile_scraper.py

```diff
@@ -0,0 +1,25 @@
+import pytest
+from bfex.components.scraper.profile_scraper import ProfileScraper
+from bfex.components.scraper.scraper_type import ScraperType
+
+base_url = "https://www.ualberta.ca/science/about-us/contact-us/faculty-directory/"
+
+
+class TestScraper():
```

Comment:
Can this file and class be called test profile scrapper?

Reply:
Whoops. Fixed it.